.travis.yml

```diff
@@ -0,0 +1,43 @@
+language: ruby
+sudo: false
+addons:
+  apt:
+    packages:
+    - libgmp-dev
+    - libmpfr-dev
+    - libmpc-dev
+    - binutils-dev
+rvm:
+ - 1.9.3
+ - 2.0
+ - 2.1
+
+install:
+  - export RUBY_SOURCE_DIR=`pwd`
+  - export TEST_CPP="no"
+
+  - git clone https://github.com/symengine/symengine symengine-cpp
+  - cd symengine-cpp
+  - git checkout 8537c67507fae9b1d8d575d322c6ade4f1a59297
```

Comment:
I see. Correct me if I am wrong. This seems to have the latest stable commit. I was wondering if we could use a tag. That would make this more clear, or a comment will also do.